src/main/scala/fs2/Process1.scala

```diff
@@ -104,6 +104,12 @@ object process1 {
   def scan1[F[_],I](f: (I, I) => I)(implicit F: NotNothing[F]): Handle[F,I] => Pull[F,I,Handle[F,I]] =
     Pull.receive1 { case o #: h => scan(o)(f).apply(h) }
 
+  /** Writes the sum of all input elements, or zero if the input is empty. */
+  def sum[F[_],I:Numeric](implicit F: NotNothing[F]): Handle[F,I] => Pull[F,I,Handle[F,I]] = {
```

Comment:
we use this a lot:

    ps.foldMap(functionToMyType)

versus

    ps.map(functionToMyType).fold(Monoid[MyType].zero)(Monoid[MyType].append)

it is very easy to provide an implicit FooMonoidToFs2Monoid to convert between an existing library of implementations, rather than needing to add the syntax to add `foldMap` or `fold*Monoid` variants per external library.